src/App.js

```diff
@@ -4,6 +4,7 @@ import "bootstrap/dist/css/bootstrap.min.css"
 import CreateTodo from "./components/create-todo.component";
 import SignIn from './components/signIn.component';
 import SignUp from './components/signUp.component';
+import ForgotPassword from './components/forgotPassword.component';
 import TodosList from './components/homePage.component';
```

Comment:
Is there a ticket in the backlog for cleaning up all the "ToDo" related sample code?  I've noticed a few PRs that replaced some of that sample code, but it seems we still have a fair amount of it lingering around.

Reply:
We can create a ticket to clean up the "ToDo"s!